src/functions/components/FunctionCard.tsx

```diff
@@ -11,17 +11,18 @@ import {
   FlexBox,
 } from '@influxdata/clockface'
 import FunctionContextMenu from 'src/functions/components/FunctionContextMenu'
-import LastRunFunctionStatus from 'src/functions/components/LastRunFunctionStatus'
 
 // utils
 import {getOrg} from 'src/organizations/selectors'
 
 interface Props {
   id: string
   name: string
+  url?: string
```

Comment:
instead of extending this props interface, you have access to the id and the function list from the context, and it seems like you've moved the functions to a hash, so why not
```
... ({ id }) => {
const {functionList} = useContext(FunctionListContext)
const { name, url, description } = functionList[id]
...
```
so that you can decouple the needs of this child component from the interface defined in the parent component. the parent shows a list, so it's always going to have some sense of "the order of things in a list" or of the list itself, but it shouldn't need to know that the items within the list require a name or url or description

Reply:
getting name description and url from context now